src/applications/facility-locator/actions/index.js

```diff
@@ -275,6 +275,77 @@ export const genBBoxFromAddress = query => {
               placeName: features[0].place_name,
               placeType: features[0].place_type[0],
             },
+            searchArea: null,
+          },
+        });
+      })
+      .catch(_ => {
+        dispatch({ type: GEOCODE_FAILED });
+        dispatch({ type: SEARCH_FAILED, error: { type: 'mapBox' } });
+      });
+  };
+};
+
+/**
+ * Calculates a human readable location (address, zip, city, state) updated
+ * from the coordinates center of the map
+ */
+export const genSearchAreaFromCenter = query => {
+  const { lat, lng } = query;
+  return dispatch => {
+    const types = ['place', 'region', 'postcode', 'locality'];
+    mbxClient
+      .reverseGeocode({
+        countries: ['us', 'pr', 'ph', 'gu', 'as', 'mp'],
```

Comment:
Can we put the country list in a cont and us it with the call to `forwardGeocode`?